olive/passes/onnx/conversion.py

```diff
@@ -119,6 +120,7 @@ def _initialize(self):
     def _run_for_config(self, model: PyTorchModel, config: Dict[str, Any], output_model_path: str) -> ONNXModel:
         pytorch_model = model.load_model()
         pytorch_model.eval()
+        pytorch_model.to(Device.CPU)
```

Comment:
better to add unit test

Reply:
Hard to directly test from unit tests. Maybe let us kick off the e2e tests for gpu in next pr when the gpu-cluster is ready.